zyte_spider_templates_project/settings.py

```diff
@@ -6,28 +6,28 @@
 BOT_NAME = "zyte_spider_templates_project"
 
 SPIDER_MODULES = [
-    "zyte_spider_templates.spiders",
     "zyte_spider_templates_project.spiders",
 ]
 NEWSPIDER_MODULE = "zyte_spider_templates_project.spiders"
 
-CLOSESPIDER_TIMEOUT_NO_ITEM = 900
 ADDONS = {
+    "zyte_spider_templates.Addon": 100,
```

Comment:
I wonder if we should put it after the other add-ons, as this add-on probably expects scrapy-zyte-api to be configured already, which the add-on below does. Maybe that’s why you ended up adding the download handler manually?